Integrations/KennaV2/KennaV2.py

```diff
@@ -0,0 +1,480 @@
+                    lst_inner = []
+                    for in_raw in inner_raw:
+                        for inner_wanted_key, inner_actual_key in zip(wanted_key[1:], actual_key[1:]):
+                            inner_dict.update({inner_wanted_key: in_raw.get(inner_actual_key)})
+                        lst_inner.append(inner_dict)
+                    context.update({wanted_key[0]: lst_inner})
+            else:
+                context.update({wanted_key: raw.get(actual_key)})
+        context_list.append(context)
+    return context_list
+
+
+# ----------------------------------------- Commands Functions ---------------------------------------------------------
+
+
+def search_vulnerabilities(client: Client, args: dict) -> Tuple[str, Dict[str, Any], List[Dict[str, Any]]]:
+    """Search vulnerability command.
+    Args:
+        client:  BaseClient which connects to api
+        args: arguments for the request
+    Returns:
+        Human Readable
+        Entry Context
+        Raw Data
+    """
+    url_suffix = '/vulnerabilities/search'
+    human_readable = []
+    context: Dict[str, Any] = {}
+    params = {
+        'id' + '[]': args.get('id'),
+        'top_priority' + '[]': args.get('top-priority'),
+        'min_risk_meter_score': args.get('min-score'),
+        'status' + '[]': args.get('status')
+    }
+    vulnerability_list = connect_api(client=client, message='GET', suffix=url_suffix,
+                                     params=params).get('vulnerabilities')
+    if vulnerability_list:
+        wanted_keys = ['AssetID', ['Connectors', 'DefinitionName', 'ID', 'Name', 'Vendor'], 'CveID', 'FixID',
+                       'ID', 'Patch',
+                       'RiskMeterScore', ['ScannerVulnerabilities', 'ExternalID', 'Open', 'Port'], 'Score',
+                       'Severity',
+                       'Status', 'Threat', 'TopPriority',
+                       ['ServiceTicket', 'DueDate', 'ExternalIdentifier', 'Status', 'TicketType']]
+        actual_keys = ['asset_id', ['connectors', 'connector_definition_name', 'id', 'name', 'vendor'], 'cve_id',
+                       'fix_id',
+                       'id', 'patch', 'lisk_meter_score',
+                       ['scanner_vulnerabilities', 'external_unique_id', 'open', 'port'],
+                       'score', 'severity', 'status', 'threat', 'top_priority',
+                       ['service_ticket', 'due_date', 'external_identifier', 'status', 'ticket_type']]
+
+        context_list = create_dict(vulnerability_list, wanted_keys, actual_keys)
+        for lst in vulnerability_list:
+            human_readable.append({
+                'id': lst.get('id'),
+                'Name': lst.get('cve_id'),
+                'Score': lst.get('risk_meter_score')
+            })
+        context = {
+            'Kenna.Vulnerabilities(val.ID === obj.ID)': context_list
+        }
+        human_readable_markdown = tableToMarkdown('Kenna Vulnerabilities', human_readable)
```

Comment:
we can order the columns in the HR, and remove nulls. please do that in all func occurrences.
thumb rule is to choose 4-6 headers for each function.